docs/pages/expokit/expokit.md

```diff
@@ -109,74 +109,77 @@ with
 
 If upgrading from SDK 32 or below:
```

Comment:
It's probably out of this PR scope, but we should remove those instructions completely, since they are referring to the few years old releases.

Reply:
Do you mean delete the ExpoKit docs from the archive section?

Reply:
Oh, good reminder, I have forgot that those pages are a part of archive/deprecated section. In this case we probably should leave those pages as-is. 🙂